_i18n/de/resources/moneropedia/openalias.md

```diff
@@ -1,19 +1,19 @@
 ---
-terms: ["OpenAlias"]
-summary: "a standard that allows you to use an email or domain syntax to pay someone instead of an address, eg. donate@getmonero.org or donate.getmonero.org"
+terms: ["OpenAlias", "OpenAlias-Adresse"]
+summary: "Ein Standard, der es ermöglicht, zur Zahlung eine Mailadresse oder einen Domainsyntax (z.B. donate@getmonero.org oder donate.getmonero.org) anstelle einer komplizierten Adresse zu verwenden"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-The Monero Core Team released a standard called OpenAlias which permits much more human-readable addresses and "squares" the Zooko's triangle. OpenAlias can be used for any cryptocurrency and is already implemented in Monero, Bitcoin (in latest Electrum versions) and HyperStake.
+Das Monero-Core-Team hat mit der sogenannten OpenAlias einen Standard veröffentlicht, der Adressen mit besserer visueller Lesbarkeit erlaubt und Zookos Dreieck "quadriert". OpenAlias kann bei jeder Kryptowährung verwendet werden und ist bisher in Monero, Bitcoin (den letzten Electrum-Versionen) und HyperStake implementiert.
 
-OpenAlias seeks to provide a way to simplify aliasing amidst a rapidly shifting technology climate. Users are trying to cross the bridge to private and cryptographically secure infrastructure and systems, but many of them have just barely started remembering the email addresses of their friends and family.
+OpenAlias bemüht sich, inmitten eines sich rapide ändernden Technologieklimas einen Weg zu öffnen, um Aliasing zu vereinfachen. Nutzer versuchen, eine Brücke zu privater und kryptografisch sicherer Infrastruktur und sicheren Systemen zu schlagen, dabei haben viele aber gerade erst damit begonnen, sich die E-Mail-Adressen ihrer Freunde und Familien zu merken.
 
-As part of the ongoing development of the Monero cryptocurrency project, we asked ourselves: how can we simplify payments for users unfamiliar with cryptocurrency? Monero stealth addresses are at least 95 characters long - memorizing them is not an option, and asking someone to send a payment to <95-character-string> is only going to lead to confusion.
+Im Zuge der kontinuierlichen Entwicklung des Monero-Kryptowährungsprojekts haben wir uns gefragt: Wie können wir Zahlungen für Nutzer vereinfachen, die mit Kryptowährungen nicht vertraut sind? Monero-Schattenadressen sind mindestens 95 Zeichen lang - sich diese zu merken ist keine Option, und jemanden darum zu bitten, eine Zahlung an eine <95stellige Zeichenkette> zu senden, würde lediglich Verwirrung stiften.
```

Comment:
```suggestion
Im Zuge der kontinuierlichen Entwicklung des Monero-Kryptowährungsprojekts haben wir uns gefragt: Wie können wir Zahlungen für Nutzer vereinfachen, die mit Kryptowährungen nicht vertraut sind? Monero Adressen sind mindestens 95 Zeichen lang - sich diese zu merken ist keine Option, und jemanden darum zu bitten, eine Zahlung an eine 95-stellige Zeichenkette zu senden, würde lediglich Verwirrung stiften.
```

"Schattenadressen"... why mention "stealth addresses".. wouldn't "Monero addresses" do, while also being easier to translate?

Reply:
@erciccione after reading this again I think "stealth addresses" is plain wrong in this context. As far as I understand, the user never gets in direct contact with "stealth addresses". They are merely generated by the sender's wallet and put on the blockchain to hide the real address of the receiver. No user ever has to "memorize a stealth address" like the English text says.

Reply:
I see the confusion the term can create, but we have to keep the original "stealth address" phrasing reguardless, as explained above. If it's wrong we can open another PR to fix it, but that's out of scope here.